atsign-foundation/at_app_cli
# Fix build test

**- What I did**

GitHub actions was not playing nice with my original build scripts, so I've fixed them here.
- Updated analysis options for each package to prevent `dart analyze` from failing unexpectedly
  - Commonly due to the `example` folder which is never bootstrapped with `pub get` in the tests
- Updated each build script to use melos' built in `$MELOS_ROOT_PATH` instead of using `$BASH_SOURCE`
- Removed bash redirections to stdin and array instantiations (which were causing the tests to fail)

**- How I did it**

Local changes, and test running the actions on my fork.

**- How to verify it**

See the last test run [here](https://github.com/XavierChanth/at_app/actions/runs/1878332897)

**- Description for the changelog**
Fix build test
